packages/components/src/whats-new/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Guide, GuidePage } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
```

Comment:
We should use `react-i18n` here with the `useI18n` hook if we can.

Reply:
> If at all possible it would be nice to go ahead and add this in TypeScript rather than adding another regular JS file to be migrated later on. However, I know that this project has already gone through so many different iterations, so I understand if there isn't time for that. If you decide to do it and run into any trouble feel free to drop a ping for help.

@sarayourfriend do you have any objections to me going with JS for this iteration? I haven't learned TypeScript yet, but it's on my near-term todo list. I'll commit to migrating it while I go through a TS course.

Reply:
Sounds good to me! If you need help, there's a #typescript channel in Slack full of very friendly folks eager to help 😀